### PR TITLE
Capability tests

### PR DESCRIPTION
**Pull request by PhysicsJon**
- [ ] Adding Epics
- [ ] Updating Appear on Preview
- [ ] Finishing Capability display test

### CODE REVIEW

File: src/MarketingPageAcceptanceTests.Steps/Steps/Authority/Capabilities/DisplaySolutionsCapabilities.cs [hunk 1]
```diff
@@ -31,31 +33,67 @@ public void GivenThatCapabilitiesHaveBeenProvidedForTheSolution()
...
         {
-            _context.Pending();
+            _test.pages.Capabilities.GetCapabilityDescriptions().Should().HaveCount(solutionCaps.Count());
         }
         
         [Then(@"'(.*)' Capability URL")]
         public void ThenCapabilityURL(string p0)
         {
-            _context.Pending();
+            _test.pages.Capabilities.GetCapabilityUrls().Should().HaveCount(solutionCaps.Count());
```
Comment (talawson05) [inline src/MarketingPageAcceptanceTests.Steps/Steps/Authority/Capabilities/DisplaySolutionsCapabilities.cs]: it looks like you've took this step out of the feature file, so do you need this step? And are we covering all the ACs without it?

File: src/MarketingPageAcceptanceTests.Steps/Steps/Authority/Capabilities/DisplaySolutionsCapabilities.cs [hunk 2]
```diff
@@ -31,31 +33,67 @@ public void GivenThatCapabilitiesHaveBeenProvidedForTheSolution()
...
         {
-            _context.Pending();
+            _test.pages.Capabilities.GetCapabilityVersions().Should().HaveCount(solutionCaps.Count());
         }
         
         [Then(@"Capability Description")]
         public void ThenCapabilityDescription()
         {
-            _context.Pending();
+            _test.pages.Capabilities.GetCapabilityDescriptions().Should().HaveCount(solutionCaps.Count());
```
Comment (talawson05) [inline src/MarketingPageAcceptanceTests.Steps/Steps/Authority/Capabilities/DisplaySolutionsCapabilities.cs]: it looks like you've took this step out of the feature file, so do you need this step? And are we covering all the ACs without it?

File: src/MarketingPageAcceptanceTests.Steps/Steps/Authority/Capabilities/DisplaySolutionsCapabilities.cs [hunk 3]
```diff
@@ -31,31 +33,67 @@ public void GivenThatCapabilitiesHaveBeenProvidedForTheSolution()
...
         public void ThenCapabilityURL(string p0)
         {
-            _context.Pending();
+            _test.pages.Capabilities.GetCapabilityUrls().Should().HaveCount(solutionCaps.Count());
         }
         
         [Then(@"the Capability URL will direct to a page that contains more content about the Capability")]
         public void ThenTheCapabilityURLWillDirectToAPageThatContainsMoreContentAboutTheCapability()
         {
             _context.Pending();
```
Comment (talawson05) [inline src/MarketingPageAcceptanceTests.Steps/Steps/Authority/Capabilities/DisplaySolutionsCapabilities.cs]: it looks like you've took this step out of the feature file, so do you need this step? And are we covering all the ACs without it?

File: src/MarketingPageAcceptanceTests.Steps/Steps/Authority/Capabilities/DisplaySolutionsCapabilities.cs (hunk 1 above)
Comment (PhysicsJon) [reply src/MarketingPageAcceptanceTests.Steps/Steps/Authority/Capabilities/DisplaySolutionsCapabilities.cs]: I'm going to be adding them back into the tests when I find a decent method of doing it

File: src/MarketingPageAcceptanceTests.Steps/Steps/Authority/Capabilities/DisplaySolutionsCapabilities.cs (hunk 2 above)
Comment (PhysicsJon) [reply src/MarketingPageAcceptanceTests.Steps/Steps/Authority/Capabilities/DisplaySolutionsCapabilities.cs]: This is still in the tests

File: src/MarketingPageAcceptanceTests.Steps/Steps/Authority/Capabilities/DisplaySolutionsCapabilities.cs (hunk 3 above)
Comment (PhysicsJon) [reply src/MarketingPageAcceptanceTests.Steps/Steps/Authority/Capabilities/DisplaySolutionsCapabilities.cs]: I'm going to be adding them back into the tests when I find a decent method of doing it

**Pull request merged**